cmd/gcs-sidecar/main.go

```diff
@@ -198,23 +225,25 @@ func main() {
 		logrus.WithError(err).Errorf("error dialing hcsshim external bridge")
 		return
 	}
-
-	// TODO CLEANUP (kiashok/Mahati):
-	// Finalize on initial policy state.
-	// Note: gcs-sidecar can be used for non-confidentail hyperv wcow
-	// as well. So we do not always want to initialize an open policy
-	// by default.
+	// gcs-sidecar can be used for non-confidentail hyperv wcow
+	// as well. So we do not always want to check for initialPolicyStance
 	var initialEnforcer securitypolicy.SecurityPolicyEnforcer
-	initialPolicyStance := "allow"
-	switch initialPolicyStance {
-	case "allow":
-		initialEnforcer = &securitypolicy.OpenDoorSecurityPolicyEnforcer{}
-		logrus.Tracef("initial-policy-stance: allow")
-	case "deny":
-		initialEnforcer = &securitypolicy.ClosedDoorSecurityPolicyEnforcer{}
-		logrus.Tracef("initial-policy-stance: deny")
-	default:
-		logrus.Error("unknown initial-policy-stance")
+	// TODO Cleanup(just for dev):
+	*confidentialWCOW = true
+	if *confidentialWCOW {
+		if *initialPolicyStance == "" {
```

Comment:
I don't think this should be the case. The policy should either be explicitly allowed or denied.